### PR TITLE
Update social and links

### PR DESCRIPTION
**Pull request by Yushi5058**
## Changes proposed

- Add a link to my portfolio website
-  Remove codewars social link
-  Add protonmail social link

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4885"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/4885/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](http://discord.eddiehub.org). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

**Review by priyankarpal — APPROVED** [review]
LGTM  🎉

**Review by iamthecloverly — DISMISSED** [review]

File: data/Yushi5058.json [hunk 1]
```diff
@@ -5,13 +5,18 @@
...
       "icon": "FaGithub"
+    },
+    {
+      "name": "Let's get in touch",
+      "url": "https://yushi5058.github.io/",
+      "icon": "FaLink"
     }
   ]
-}
+}
```
Comment (Panquesito7) [inline data/Yushi5058.json:22]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — DISMISSED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**